pkg/pmem-device-manager/pmd-util.go

```diff
@@ -75,5 +78,5 @@ func waitDeviceAppears(dev *PmemDeviceInfo) error {
 			i, dev.Path, retryStatTimeout)
 		time.Sleep(retryStatTimeout)
 	}
-	return fmt.Errorf("device %s did not appear after multiple retries", dev.Path)
+	return fmt.Errorf("%w(%s)", ErrDeviceNotReady, dev.Path)
```

Comment:
The Go idiom is: "%s: %w" (https://blog.golang.org/go1.13-errors).

Reply:
Done.